[2024-09-17T23:23:40.797+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-17T06:15:00+00:00 [queued]>
[2024-09-17T23:23:40.813+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-17T06:15:00+00:00 [queued]>
[2024-09-17T23:23:40.814+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-09-17T23:23:40.844+0000] {taskinstance.py:1382} INFO - Executing <Task(SSHOperator): raw_ingestion> on 2024-09-17 06:15:00+00:00
[2024-09-17T23:23:40.851+0000] {standard_task_runner.py:57} INFO - Started process 1237 to run task
[2024-09-17T23:23:40.857+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'raw_ingestion', 'raw_ingestion', 'scheduled__2024-09-17T06:15:00+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/raw_ingestion.py', '--cfg-path', '/tmp/tmpn68cjhtq']
[2024-09-17T23:23:40.858+0000] {standard_task_runner.py:85} INFO - Job 82: Subtask raw_ingestion
[2024-09-17T23:23:41.083+0000] {task_command.py:416} INFO - Running <TaskInstance: raw_ingestion.raw_ingestion scheduled__2024-09-17T06:15:00+00:00 [running]> on host airflow-worker
[2024-09-17T23:23:41.432+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='raw_ingestion' AIRFLOW_CTX_TASK_ID='raw_ingestion' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T06:15:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-17T06:15:00+00:00'
[2024-09-17T23:23:41.434+0000] {ssh.py:141} INFO - Creating ssh_client
[2024-09-17T23:23:41.436+0000] {ssh.py:114} INFO - ssh_hook is not provided or invalid. Trying ssh_conn_id to create SSHHook.
[2024-09-17T23:23:41.494+0000] {base.py:73} INFO - Using connection ID 'spark_ssh_default' for task execution.
[2024-09-17T23:23:41.495+0000] {ssh.py:300} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2024-09-17T23:23:41.564+0000] {transport.py:1893} INFO - Connected (version 2.0, client OpenSSH_8.9p1)
[2024-09-17T23:23:41.892+0000] {transport.py:1893} INFO - Authentication (password) successful!
[2024-09-17T23:23:41.893+0000] {ssh.py:478} INFO - Running command: export SPARK_HOME=/usr/local/spark; /opt/conda/bin/spark-submit --master local[*] --executor-memory 2g /opt/shared/scripts/raw_ingestion.py
[2024-09-17T23:23:53.227+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 178, in execute
    result = self.run_ssh_client_command(ssh_client, self.command, context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/operators/ssh.py", line 163, in run_ssh_client_command
    exit_status, agg_stdout, agg_stderr = self.hook.exec_ssh_client_command(
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/ssh/hooks/ssh.py", line 549, in exec_ssh_client_command
    raise AirflowException("SSH command timed out")
airflow.exceptions.AirflowException: SSH command timed out
[2024-09-17T23:23:53.233+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=raw_ingestion, task_id=raw_ingestion, execution_date=20240917T061500, start_date=20240917T232340, end_date=20240917T232353
[2024-09-17T23:23:53.267+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 82 for task raw_ingestion (SSH command timed out; 1237)
[2024-09-17T23:23:53.288+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-17T23:23:53.352+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
